src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Group.java

```diff
@@ -39,7 +39,7 @@
 
 	@NotNull
 	@Column(name = "group_number")
-	private long groupNumber;
+	private Long groupNumber;
```

Comment:
I changed this to a `Long` because a primitive defaults to `0` and I'm not sure if that is desired behavior